src/backend/vulkan/Cargo.toml

```diff
@@ -15,6 +15,7 @@ edition = "2018"
 [features]
 default = []
 use-rtld-next = ["libc"]
+use-openxr = ["openxr", "once_cell"]
```

Comment:
Initially went with "use-openxr" because "openxr" clashed with the crate with same name, but I guess that's doable as a name as well.

I like the 'xr', can make the change. Although, that might make the name ambiguous with other possible xr implementations in the future.